Scripts/RemediationScripts/Remediate-SetDBForPostgreSQLSingleServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,847 @@
+<###
+# Overview:
+    This script is used to set minimium required TLS version and enable ssl enforcement for Azure Database for PostgreSQL single server in a Subscription.
+# Control ID:
+    Azure_DBforPostgreSQL_DP_Use_Secure_TLS_Version
+# Display Name:
+    Use approved version of TLS for Azure Database for PostgreSQL - single Server.
+    
+# Prerequisites:    
+    Contributor or higher priviliged role on the Azure Database for PostgreSQL single server(s) is required for remediation.
+# Steps performed by the script:
+    To remediate:
+        1. Validating and installing the modules required to run the script and validating the user.
+        2. Get the list of Azure Database for PostgreSQL single server(s) in a Subscription that have server parameter minimaltlsversion set as versions less than minimum required TLS version or parameter SslEnforcement is set to OFF .
+        3. Back up details of Azure Database for PostgreSQL single server(s) that are to be remediated.
+        4. Set supported minimum required TLS version by updating parameter minimaltlsversion as minimum required TLS version and enable SslEnforcement by updating server parameter SslEnforcement to ON for Azure Database for PostgreSQL single server(s).
+    To roll back:
+        1. Validate and install the modules required to run the script and validating the user.
+        2. Get the list of Azure Database for PostgreSQL single server(s) in a Subscription, the changes made to which previously, are to be rolled back.
+        3. Set the server parameter minimaltlsversion to original value and set server parameter SslEnforcement to ON as per input file.
+# Instructions to execute the script:
+    To remediate:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to set supported TLS version for Azure Database for PostgreSQL single server(s) in the Subscription. Refer `Examples`, below.
+    
+    After script execution: 
+        As minimaltlsversion is Static parameter that needs server restart for updated value to take effect, server restart is recommended to be done seperately after script execution.
```

Comment:
Does it really require restart? Did we verify?

Reply:
it doesn't required restart, removed.